recipe/meta.yaml

```
@@ -28,26 +28,27 @@ requirements:
 
   run:
     - python
-    - apispec >=3.3,<4
+    - apispec >=6.0.0,<7
     - colorama >=0.3.9,<1
     - click >=8,<9
     - email_validator >=1.0.5,<2
```

```suggestion
    - email_validator >=1.0.5
```

https://github.com/dpgaspar/Flask-AppBuilder/blob/v4.6.2/setup.py#L51